snow/engine/enginetest/sender.go

```diff
@@ -15,18 +16,18 @@ import (
 )
 
 var (
-	_ common.Sender    = (*SenderTest)(nil)
-	_ common.AppSender = (*FakeSender)(nil)
+	_ common.Sender    = (*Sender)(nil)
+	_ common.AppSender = (*SenderStub)(nil)
 
 	errSendAppRequest  = errors.New("unexpectedly called SendAppRequest")
 	errSendAppResponse = errors.New("unexpectedly called SendAppResponse")
 	errSendAppError    = errors.New("unexpectedly called SendAppError")
 	errSendAppGossip   = errors.New("unexpectedly called SendAppGossip")
 )
 
-// SenderTest is a test sender
-type SenderTest struct {
-	T require.TestingT
+// Sender is a test sender
+type Sender struct {
+	T *testing.T
```

Comment:
Really it just jumped out at me as a bit of a code smell because the `require.TestingT` interface is redundant due to `testing.TB` existing. But I chose the struct over the interface because of consistency.